dev/io.openliberty.concurrent.internal/resources/io/openliberty/concurrent/internal/resources/CWWKCMessages.nlsprops

```diff
@@ -45,3 +45,7 @@ CWWKC1203.duplicate.context.useraction=Update the application, libraries, or bot
 CWWKC1204.not.serializable=CWWKC1204E: Cannot create a serializable contextual proxy that propagates thread context {0} which is not serializable.
 CWWKC1204.not.serializable.explanation=Third-party thread context types are not compatible with serialization and cannot be configured to be propagated for serializable contextual proxies.
 CWWKC1204.not.serializable.useraction=Update the application to request a non-serializable contextual proxy or ensure that no third-party context types are configured to be propagated.
+
+CWWKC1205.qualifier.class.not.found=CWWKC1205E: Qualifier annotation {0}, which is defined by the application deployment descriptor, could not be loaded by application classloader.
+CWWKC1205.qualifier.class.not.found.explanation=The <qualifier> element(s) configured via application.xml or web.xml contained a class name which could not be loaded.
+CWWKC1205.qualifier.class.not.found.useraction=Verify the <qualifier> element(s) were configured with a fully qualified class name, and the class is present within the application.
```

Comment:
Suggested rewrite:
Verify that the qualifier elements are configured with a fully qualified class name, and that the class is present within the application.